### ISSUE TITLE
Data Quality - Test on complex types

### ISSUE THREAD

**Issue by pmbrull** [open-metadata/OpenMetadata#3228]
There are close to no metrics supported right now for complex types, such as JSON or ARRAY. However, from the UI we can configure all tests.

Most of them will be aborted:

![image](https://user-images.githubusercontent.com/35870520/157060449-7f92cbe2-5daa-45ce-9816-e4346e9a2580.png)

@harshach, would it make sense to disable some tests in the UI for these types? Might avoid having tests running that will always fail 🤔

**Comment by harshach** [comment]
@pmbrull yes we should disable and show a text that complex types are not supported. cc @ShaileshParmar11 @Sachin-chaurasiya

**Comment by pmbrull** [comment]
Hi @Sachin-chaurasiya @ShaileshParmar11, as part of this ticket we should also review the tests shown for all the types:

## Numeric
- Unique
- Not Null
- Not In Set
- Values Between
- Missing Count

## String
- Unique
- Not Null
- Not in Set
- Values Lengths Between
- Missing Count
